multimodal/src/autogluon/multimodal/models/mmdet_image.py

```diff
@@ -68,7 +68,10 @@ def __init__(
         self.checkpoint_name = checkpoint_name
         self.config_file = config_file
         self.classes = classes
-        self.frozen_layers = frozen_layers
+        # Based on our offline benchmarking results, instead of freezing layers,
+        # Setting backbone to a smaller learning rate achieves better results,
+        self.frozen_layers = []
+        self.backbone_layers = frozen_layers
```

Comment:
1. Assigning `frozen_layers` to `backbone_layers` looks confusing. How about having a new attribute `backbone_layers`?
2. This change also deprecates the previous functionality of supporting freezing layers. Is this deprecation a good choice? Maybe we still support freezing layers as a choice.

Reply:
Normally frozen_layers and backbone_layers should refer to the same layers. It's just are we freezing the backbone layers or using a small lr. Adding a new hyperparameter seems redundant.

Reply:
I think we need to make names clear to improve code readability. Otherwise, others probably can't understand them correctly. If we no longer support freezing layers, then we should remove it to reduce confusion.

Reply:
Sent an up date to remain frozen layers, and set up backbone to have low lr for dino